pkg/agent/route/route_windows.go

```diff
@@ -330,9 +365,96 @@ func (c *Client) UnMigrateRoutesFromGw(route *net.IPNet, linkName string) error
 
 // Run is not supported on Windows and returns immediately.
 func (c *Client) Run(stopCh <-chan struct{}) {
+	klog.Infof("Starting netNatStaticMapping and route sync, with sync interval %v", syncInterval)
+	wait.Until(c.syncIPInfra, syncInterval, stopCh)
+}
+
+// syncIPInfra is idempotent and can be safely called on every sync operation.
+func (c *Client) syncIPInfra() {
+	if err := c.syncRoute(); err != nil {
+		klog.Errorf("Failed to sync route: %v", err)
+	}
+
+	if c.proxyAll {
+		if err := c.syncNetNatStaticMapping(); err != nil {
+			klog.Errorf("Failed to sync netNatStaticMapping: %v", err)
+		}
+	}
+
+	klog.V(3).Infof("Successfully synced Node netNatStaticMapping and route")
 }
 
-func (c *Client) listRoutes() (map[string]*util.Route, error) {
+func (c *Client) syncRoute() error {
+	restoreRoute := func(route *util.Route) bool {
+		if err := util.ReplaceNetRoute(route); err != nil {
+			klog.Errorf("Failed to add route to the gateway: %v", err)
+			return false
+		}
+		return true
+	}
+	c.nodeRoutes.Range(func(_, v interface{}) bool {
+		route := v.(*util.Route)
+		return restoreRoute(route)
+	})
+	if c.proxyAll {
+		c.serviceRoutes.Range(func(_, v interface{}) bool {
+			route := v.(*util.Route)
+			return restoreRoute(route)
+		})
+	}
+	// These routes are installed automatically by the kernel when the address is configured on
+	// the interface. If these routes are deleted manually by mistake, we restore them.
+	gwAutoconfRoutes := []*util.Route{
+		{
+			LinkIndex:         c.nodeConfig.GatewayConfig.LinkIndex,
+			DestinationSubnet: util.NewIPNet(c.nodeConfig.GatewayConfig.IPv4),
+			GatewayAddress:    c.nodeConfig.GatewayConfig.IPv4,
+			RouteMetric:       util.MetricDefault,
+		},
+		{
+			LinkIndex:         c.nodeConfig.GatewayConfig.LinkIndex,
+			DestinationSubnet: c.nodeConfig.PodIPv4CIDR,
+			GatewayAddress:    c.nodeConfig.GatewayConfig.IPv4,
+			RouteMetric:       util.MetricDefault,
+		},
+	}
+
+	for _, route := range gwAutoconfRoutes {
+		restoreRoute(route)
+	}
+	return nil
+}
+
+func (c *Client) syncNetNatStaticMapping() error {
+	if err := util.NewNetNat(antreaNatNodePort, virtualNodePortDNATIPv4Net); err != nil {
+		return err
+	}
```

Comment:
`util.NewNetNat` produces logs, periodical calling it may lead to log spam

Reply:
I'll set log level in `util.NewNetNat`